internal/pkg/cli/dialog/use_template.go

```diff
@@ -109,6 +111,10 @@ func (d *useTmplDialog) askInputs(inputs *input.Inputs) error {
 		if result, err := inputDef.Available(d.inputsValues); err != nil {
 			return err
 		} else if !result {
+			// Input is hidden, add default/empty value
+			if err := d.addInputValue(inputDef.DefaultOrEmpty(), inputDef, false); err != nil {
+				return err
+			}
```

Comment:
Ukazalo sa, ze ak je `showIf` vyhodnotene ako `false`, tak je lepsie aby bola v `Jsonnet` dostupna prazdna/default hodnota, ako ked `Input` vobec neexistuje a spadne to ... kedze nemame aktualne ako zistit, ze vstup bol ignorovany. A na pridanie dalsie funckie `InputSet` teraz nie je velmi cas.